backend/src/external/services/powerbi-api.ts

```diff
@@ -0,0 +1,112 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+
+/**
+ * This file contains all the PowerBi Types and a wrapper to access the REST API endpoints
+ */
+
+// A wrapper for all the REST API endpoints
+export const Api = {
+  /** https://learn.microsoft.com/en-us/rest/api/power-bi/dashboards/get-dashboard-in-group#dashboard */
+  getDashboardInGroup: (
+    url: DashboardInGroup_Url,
+    config?: AxiosRequestConfig,
+  ): Promise<AxiosResponse<Dashboard>> =>
+    axios.get<Dashboard>(
+      `https://api.powerbi.com/v1.0/myorg/groups/${url.workspaceId}/dashboards/${url.dashboardId}`,
```

Comment:
[http](https://api.powerbi.com/) read from the same config

Reply:
I also made api.powerbi.com configurable. I think that's what you mean.